openmetadata-service/pom.xml

```diff
@@ -605,6 +605,23 @@
       <version>1.13</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
+      <artifactId>owasp-java-html-sanitizer</artifactId>
+      <version>20211018.2</version> 
```

Comment:
put the versions in one place and instead of adding this into opnemetadata-service. We should clean this up and add a separate module called openmetadata-apps. 
App interfaces and others can live in the openmetadata-spec and implementation can go to openmetadata-apps

Reply:
Noted.